packages/trader/src/AppV2/Components/ClosedMarketMessage/closed-market-message.tsx

```diff
@@ -37,9 +37,10 @@ const ClosedMarketMessage = observer(() => {
     const [when_market_opens, setWhenMarketOpens] = React.useState<TWhenMarketOpens>({} as TWhenMarketOpens);
     const [time_left, setTimeLeft] = React.useState(calculateTimeLeft(when_market_opens?.remaining_time_to_open));
     const [is_loading, setLoading] = React.useState(true);
+    const is_market_closed = isMarketClosed(activeSymbols, symbol);
```

Comment:
is it still reactive based on this change? since activeSymbols is not part of the dependency anymore

Reply:
On localhost looks reactive. I've added is_market_closed to array with dependencies, and it is based on activeSymbols.
Do you think it'll be better to leave activeSymbols in array? Might be, at least it was already tested

Reply:
Done, added it back just in case

Reply:
try with is_market_closed from trade-store ?

Reply:
Let me double check, because previously on localhost it was working with is_market_closed from trade-store

Reply:
Any specific edge test cases?